REQUIRE

```diff
@@ -1,3 +1,3 @@
 julia 0.4
-Compat 0.8.0
+Compat 0.9.5
```

Comment:
I can't remember, maybe it was the `Symbol` constructor?  I definitely upgrade the Compat requirement for a reason.